fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -112,6 +112,10 @@ default Table getTable(String dbName, String tblName) {
         return null;
     }
 
+    default boolean tableExists(String dbName, String tblName) {
+        return getTable(dbName, tblName) != null;
```

Comment:
it's best to use `listPartitionNames.contains` to check as default.

Reply:
Done